cdc/processor/processor.go

```diff
@@ -368,7 +368,7 @@ func (p *processor) handleTableOperation(ctx cdcContext.Context) error {
 				if opt.BoundaryTs < globalCheckpointTs {
 					log.Warn("the BoundaryTs of remove table operation is smaller than global checkpoint ts", zap.Uint64("globalCheckpointTs", globalCheckpointTs), zap.Any("operation", opt))
 				}
-				if !table.AsyncStop(opt.BoundaryTs) {
+				if !table.AsyncStop(p.changefeed.Status.ResolvedTs) {
```

Comment:
After `table.AsyncStop` is called, do we have a phase to wait the table checkpoint to reach the global resolved ts.

If not, in L388 the BoundaryTs of new table could be less than global resolved ts, and there still exists the risk that two processors write single table.